modules/end-user-guide/partials/proc_allowing-users-to-define-workspace-deployment-labels-and-annotations.adoc

```diff
@@ -73,7 +73,7 @@ To add or override annotations, use the following factory URL:
 
 [subs="+quotes"]
 ----
-https://che.openshift.io/f?url=https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml&workspaceDeploymentAnnotations=ike.A1%3Dpreference-v1%2Cike.A%3Dtest
+https://workspaces.openshift.com/#__<repository-url>__&workspaceDeploymentAnnotations=ike.A1%3Dpreference-v1%2Cike.A%3Dtest
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on f?url= see #1884

why do you change `f?url=` to non-working `#` ?